src/main/java/me/contaria/seedqueue/SeedQueue.java

```diff
@@ -71,6 +73,27 @@ public void onInitializeClient() {
         }
     }
 
+    private static void logSystemInformation() {
+        int availableProcessors = Runtime.getRuntime().availableProcessors();
+        int totalProcessors = getTotalProcessors();
+
+        LOGGER.info("System Information (Logged by SeedQueue):");
+        LOGGER.info("Operating System: {}", System.getProperty("os.name"));
+        LOGGER.info("OS Version: {}", System.getProperty("os.version"));
+        LOGGER.info("CPU: {}", System.getenv("PROCESSOR_IDENTIFIER"));
+        LOGGER.info("Java Version: {}", System.getProperty("java.version"));
+        LOGGER.info("JVM Arguments: {}", String.join(" ", java.lang.management.ManagementFactory.getRuntimeMXBean().getInputArguments()));
+        LOGGER.info("Allocated Memory (MB): {}", Runtime.getRuntime().totalMemory() / (1024 * 1024));
+        LOGGER.info("Max Memory (MB): {}", Runtime.getRuntime().maxMemory() / (1024 * 1024));
+        LOGGER.info("Free Memory (MB): {}", Runtime.getRuntime().freeMemory() / (1024 * 1024));
```

Comment:
Correct me if I'm wrong, but `Runtime#totalMemory` returns the amount of RAM used at that very moment, which isn't useful and `Runtime#freeMemory` also isn't helpful.
Max Memory seems to be Allocated Memory and if there is some kind of way to get the systems total memory through `ManagementFactory` that would be helpful information